mscore/shortcut.cpp

```diff
@@ -3969,6 +3969,9 @@ static QList<Shortcut1> loadShortcuts(QString fileLocation)
 void Shortcut::loadFromNewFile(QString fileLocation)
       {
       QList<Shortcut1> list = loadShortcuts(fileLocation);
+      if (list.length() == 0) {
+            QMessageBox::critical(0, "ERROR", "The shortcut file chosen could not load properly. Try loading another file.");
```

Comment:
At this place it is too late to give a decent error message that includes the real reason for the failure, better do it inside loadShortcuts(). Check how it is done elsewhere, e.g. when loading a style file.
Here we'd need to return early, to not set `dirty` on a failed load.
Also the error message needs to be translatable.